backend/tracim_backend/lib/search/elasticsearch_search/elasticsearch_search.py

```diff
@@ -1166,28 +1166,28 @@ def on_content_modified(self, content: Content, context: TracimContext) -> None:
         self.index_user(content.current_revision.owner, context)
 
     @hookimpl
-    def on_user_role_in_workspace_created(
+    def on_user_config_in_workspace_created(
         self,
-        role: UserRoleInWorkspace,
+        config: UserConfigInWorkspace,
```

Comment:
same comment about config

In the entire file

Reply:
done

Reply:
done